src/NuGet.Clients/VisualStudio/RestoreManager/IVsProjectReference.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace NuGet.VisualStudio
+{
+    /// <summary>
+    /// Project reference metadata
+    /// </summary>
+    [ComImport]
+    [Guid("91db043c-6766-4d2c-9885-6861457bac76")]
+    public interface IVsProjectReference
```

Comment:
This is locking the data model down pretty hard, do we need to create a new interface every time a new piece of data needs to be copied over, while maintaining this old one?